hs_app_timeseries/models.py

```diff
@@ -750,3 +1976,11 @@ def _generate_term_from_name(name):
     # first word lowercase, subsequent words start with a uppercase
     term = name_parts[0].lower() + ''.join([item.title() for item in name_parts[1:]])
     return term
+
+
+def _get_series_ids(element_class, metadata_obj):
+    series_ids = []
+    elements = element_class.objects.filter(object_id=metadata_obj.id).all()
+    for element in elements:
+        series_ids += element.series_ids
```

Comment:
Hmm, wouldn't ```series_ids += element.series_ids``` have the same effect as ```series_ids = [element.series_ids for element in elements]```? Anyway, if you think this would not meet your requirement, the current implementation is OK.